### ISSUE TITLE
Renaming TODOs

### ISSUE THREAD

**Issue by fdietze** [alstr/todo-to-issue-action#73]
I just had an idea how to handle renames:
If there is a TODO **removed and added** in the same line of the file, it should be renamed in the github issues.

Would that be possible?

**Comment by alstr** [comment]
Good suggestion. I'll look into it!

**Comment by alstr** [comment]
Hi @tami5, thanks very much. At the moment updating TODOs isn't supported but it is still something I want to add. It seems possible. I'll try and look at it soon but if anyone wants to submit a PR feel free.

**Comment by iamKunal** [comment]
@alstr Maybe the reference can be used as a unique identifier? An boolean input with `REFERENCE_AS_IDENTIFIER` can be perhaps used to link todos to issues.
If reference is an issue, maybe a new field called `identifier` be used similar to `labels` and `milestone`

 It can also be used to solve #73 and maybe even #98

**Comment by alstr** [comment]
@iamKunal Yes, that's a good idea and probably the best option at this stage. Definitely something I would like to look into.

**Comment by alstr** [comment]
Will try and look into this as part of #123. Not sure if it's best to update the issue or append to the issue at this stage.
